azure-pipelines.yml

```diff
@@ -0,0 +1,19 @@
+steps:
+- task: Gradle@2
+  displayName: Build core
+  inputs:
```

Comment:
WDYT think about pinning a specific JDK version and architecture? (`jdkVersionOption` and `jdkArchitectureOption` AIUI)?

Not because I'm expecting your build server to suddenly change without you noticing, but just to be explicit and reproducible by others 😄

Reply:
weird, I thought I set it already :) will do!

Reply:
ah, right.... different project :D 
https://github.com/reactor/BlockHound/blob/af6e0c9a17468927535490f70e8e1d23663ff1b9/.ci/azure-pipelines.yml#L17

Reply:
done (see `jdkVersionOption`)